### DISCUSSION TITLE
Streamline whitespace and comment handling

### DISCUSSION THREAD

**Discussion by oliverklee** [MyIntervals/PHP-CSS-Parser#679] [Ideas]
Which steps do we need to take in which order, and for which branches?

- [x] #672
  - [x] `main`
  - [x] `v8.x`
- [x] #674 (which will allow us to courageously refactor the parsing code)
  - [x] `main`
  - [x] `v8.x`
- [ ] ##680
  - [ ] `main`
  - [ ] `v8.x`

@ziegenberg Which other steps for which branches do you think we should take? If you comment here, I'll update the checklist accordingly.

Hopefully this discussion and checklist will help us get the moving target move a bit less, and create a shared understanding of the plan. :-)

**Comment by JakeQZ** [comment]
For cross-reference, this discussion arose from #663.
